File: benches/baselines/micro/str_build.js
// String building: 60k appends (naive concat, same as the others)
const t0 = Date.now();
let s = "";
let i = 0;
while (i < 60000) {
    s = s + "ab" + String(i % 10);
    i = i + 1;
}
const t1 = Date.now();
print_line(`RESULT ${s.length} MS ${t1 - t0}`);

File: benches/baselines/micro/str_build.lua
-- String building: 60k appends (naive concat, same as the others)
local t0 = os.clock()
local s = ""
local i = 0
while i < 60000 do
  s = s .. "ab" .. tostring(i % 10)
  i = i + 1
end
local t1 = os.clock()
print(string.format("RESULT %d MS %d", #s, math.floor((t1 - t0) * 1000)))
